packages/dds/matrix/src/test/memory/matrix.spec.ts

```diff
@@ -34,95 +34,116 @@ describe("Matrix memory usage", () => {
 		// See the comment at the top of the test suite for more details.
 	});
 
-	benchmarkMemory(
-		new (class implements IMemoryTestObject {
-			title = "Create empty Matrix";
-			minSampleCount = 500;
-
-			private localMatrix: SharedMatrix = createLocalMatrix("testLocalMatrix");
-
-			async run() {
-				this.localMatrix = createLocalMatrix("testLocalMatrix");
-			}
-		})(),
-	);
-
-	const numbersOfEntriesForTests = [100, 1000, 10_000];
-
-	numbersOfEntriesForTests.forEach((x) => {
+	describe("Detached Matrix", () => {
 		benchmarkMemory(
 			new (class implements IMemoryTestObject {
-				title = `Insert and remove ${x} columns`;
+				title = "Create empty Matrix";
+				minSampleCount = 500;
+
 				private localMatrix: SharedMatrix = createLocalMatrix("testLocalMatrix");
 
 				async run() {
-					for (let i = 0; i < x; i++) {
-						this.localMatrix.insertCols(0, 100);
-						this.localMatrix.removeCols(0, 100);
-					}
-				}
-
-				beforeIteration() {
 					this.localMatrix = createLocalMatrix("testLocalMatrix");
 				}
 			})(),
 		);
 
-		benchmarkMemory(
-			new (class implements IMemoryTestObject {
-				title = `Insert and remove ${x} rows`;
-				private localMatrix: SharedMatrix = createLocalMatrix("testLocalMatrix");
+		const numbersOfEntriesForTests = [100, 1000, 5000];
 
-				async run() {
-					for (let i = 0; i < x; i++) {
-						this.localMatrix.insertRows(0, 100);
-						this.localMatrix.removeRows(0, 100);
-					}
-				}
+		numbersOfEntriesForTests.forEach((x) => {
+			benchmarkMemory(
+				new (class implements IMemoryTestObject {
+					title = `Insert and remove a column ${x} times`;
```

Comment:
Nit: Can these `title` properties be made `readonly`?

Reply:
Done! The interface in @fluid-tools/benchmark should probably enforce that. I'll create a ticket for that.